### PR TITLE
PDP-985: Add labels / Remove network policies / Edit _helpers.tpl

### PR DESCRIPTION
**Pull request by nashtapu-tibco**

### CODE REVIEW

File: charts/oauth2-proxy-manifests/templates/_helpers.tpl [hunk 1]
```diff
@@ -40,6 +40,7 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 app.kubernetes.io/component: authentication-proxy
 app.kubernetes.io/part-of: {{ template "oauth2-proxy.name" . }}
 {{- include "oauth2-proxy.selectorLabels" . }}
+{{- include "oauth2-proxy.platformLabels" . }}
```
Comment (pagarwal-tibco) [inline charts/oauth2-proxy-manifests/templates/_helpers.tpl:43]: chart version is not updated

File: charts/dp-core-infrastructure/charts/haproxy-ingress/templates/_helpers.tpl [hunk 2]
```diff
@@ -66,6 +66,8 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 {{- /* platform specific labels. Copy paste these labels after upgrading the haproxy chart.*/}}
 platform.tibco.com/dataplane-id: {{ .Values.global.tibco.dataPlaneId }}
 platform.tibco.com/workload-type: infra
+egress.networking.platform.tibco.com/internet-all: enable
```
Comment (pagarwal-tibco) [inline charts/dp-core-infrastructure/charts/haproxy-ingress/templates/_helpers.tpl:69]: dp-core-infra chart version to be updated?

File: charts/oauth2-proxy-manifests/templates/_helpers.tpl (hunk 1 above)
Comment (nashtapu-tibco) [reply charts/oauth2-proxy-manifests/templates/_helpers.tpl:43]: I am not sure how @Stefan Nenkov is managing the chart versions, so I’ll request him in the message that we are going to send in the channel to update chart version.

File: charts/dp-core-infrastructure/charts/haproxy-ingress/templates/_helpers.tpl (hunk 2 above)
Comment (nashtapu-tibco) [reply charts/dp-core-infrastructure/charts/haproxy-ingress/templates/_helpers.tpl:69]: Updated the chart version 1.0.241

Comment (sasati-tibco) [inline charts/dp-core-infrastructure/charts/haproxy-ingress/templates/_helpers.tpl:69]: why do we need the internet all for HAProxy ?

**Pull request merged**